hack/test-cmd.sh

```diff
@@ -12,8 +12,7 @@ function cleanup()
     set +e
 
     echo "[INFO] Dumping etcd contents to ${ARTIFACT_DIR}/etcd_dump.json"
-    set_curl_args 0 1
-    curl -s ${clientcert_args} -L "${API_SCHEME}://${API_HOST}:${ETCD_PORT}/v2/keys/?recursive=true" > "${ARTIFACT_DIR}/etcd_dump.json"
+    oc get --raw "${API_SCHEME}://${API_HOST}:${ETCD_PORT}/v2/keys/?recursive=true" --config="${MASTER_CONFIG_DIR}/admin.kubeconfig" > "${ARTIFACT_DIR}/etcd_dump.json"
```

Comment:
admin.kubeconfig does not contain etcd credentials
master.etcd-client.crt/master.etcd-client.key/ca.crt are more what you want